src/plugins/visualizations/public/plugin.ts

```diff
@@ -107,13 +114,16 @@ export class VisualizationsPlugin
       VisualizationsStartDeps
     > {
   private readonly types: TypesService = new TypesService();
+  private getStartServicesOrDie?: StartServicesGetter<VisualizationsStartDeps, VisualizationsStart>;
 
   constructor(initializerContext: PluginInitializerContext) {}
 
   public setup(
-    core: CoreSetup,
+    core: CoreSetup<VisualizationsStartDeps, VisualizationsStart>,
     { expressions, embeddable, usageCollection, data }: VisualizationsSetupDeps
   ): VisualizationsSetup {
+    const start = (this.getStartServicesOrDie = createStartServicesGetter(core.getStartServices));
```

Comment:
update to the shared utility

Reply:
I've added JSDocs for `createStartServicesGetter` method.